core/DataAccess/Model.php

```diff
@@ -335,8 +334,8 @@ private function makeExistingInvalidationArrayKey($idSite, $date1, $date2, $peri
     /**
      * @param string $archiveTable Prefixed table name
      * @param int[] $idSites
-     * @param string[][] $datesByPeriodType
-     * @param Segment $segment
+     * @param Period[] $allPeriodsToInvalidate
+     * @param null|Segment $segment
```

Comment:
```suggestion
     * @param Segment|null $segment
```

It is the same, but with the usual order 😄